webapp/channels/src/components/sidebar/sidebar_category/sidebar_category_menu/index.tsx

```diff
@@ -194,6 +196,11 @@ const SidebarCategoryMenu = ({
                     />
                 )}
                 onClick={() => handleSortChannels(CategorySorting.Alphabetical)}
+                trailingElements={sortBehavior === CategorySorting.Alphabetical ? (
```

Comment:
Could we use `category.sorting` here instead of `sortBehaviour`? `category` is passed in as a prop, so I wouldn't expect to see it stored in a `useState` as well

Reply:
Makes sense. Fixed it.